static/js/bundles/app.js

```diff
@@ -10,6 +10,7 @@ import "jquery-validation";
 import "flatpickr";
 
 // Import app JS
+import "../all_messages_data";
```

Comment:
Can we move this after e.g. `setup` and `templates`?  I'm pretty sure `setup` at least was intentionally first.

Reply:
Let's see what @andersk advises.  It's possible that we just want to lazy-create the actual instance of the message list (but super early in `ui_init`.)

Reply:
I think my proposed breakup of `message_finder` will remove the need for this change.